### ISSUE TITLE
Use JDK 11 as boot JDK for Travis and Appveyor scripts

### ISSUE THREAD

**Issue by kevinrushforth** [javafxports/openjdk-jfx#159]
Shortly after JDK 11 ships, I plan to bump the minimum version of boot JDK needed to build openjfx 12 to JDK 11. Before doing that, the Appveyor and Travis CI scripts need to be updated to use a recent OpenJDK 11 EA build.

**Comment by kevinrushforth** [comment]
@brcolow Can you say how much work this will be?

**Comment by brcolow** [comment]
Shouldn't be much work at all. On Appveyor we try and use `choco` to install packages. If Java 11 is not yet available on `choco` (which I don't think it is yet) we can do a manual install as we do now:

```powershell
$openJdk10 = 'https://github.com/AdoptOpenJDK/openjdk10-nightly/releases/download/jdk-10%2B23-20180524/OpenJDK10_x64_Win_20180524.zip'
      $client.DownloadFile($openJdk10, 'C:\Users\appveyor\openjdk10.zip')
      Expand-Archive -Path 'C:\Users\appveyor\openjdk10.zip' -DestinationPath 'C:\Users\appveyor\openjdk10'
      Copy-Item -Path 'C:\Users\appveyor\openjdk10\*\' -Destination 'C:\jdk10' -Recurse -Force
```
(along with changing `JAVA_HOME`).

On Travis CI all that must be done is change `jdk: openjdk10` to `jdk: openjdk11`.

**Comment by kevinrushforth** [comment]
Good to know. We'll want to give this a try in a week or two.

**Comment by kevinrushforth** [comment]
I don't see any JDK 11 EA binaries on AdoptOpenJDK. Am I not looking in the right place? I should be able to use the ones from jdk.java.net for testing.

**Comment by johanvos** [comment]
You can take the OpenJDK 11-ea builds here: http://jdk.java.net/11/

Op do 23 aug. 2018 om 18:43 schreef Kevin Rushforth <
notifications@github.com>:

**Comment by kevinrushforth** [comment]
> You can take the OpenJDK 11-ea builds here: http://jdk.java.net/11/

Yes, that's what I plan to do. There should be no problem, although it does mean we will need to download it explicitly rather than being able to just specify `openjdk11` in the `.travis.yml` file.

**Comment by kevinrushforth** [comment]
Initial test build here: #165

**Comment by kevinrushforth** [comment]
>  it does mean we will need to download it explicitly rather than being able to just specify openjdk11 in the .travis.yml file.

I stand corrected. Specifying `openjdk11` as the JDK in `.travis.yml` does download it from the right place.

**Comment by kevinrushforth** [comment]
PR #165 is now ready for review.
